### PR TITLE
search for references from IClassFile without source

### LINKED ISSUE #1665: search for enum reference without source
Vscode does not find references on Enum when it does not have the source.

##### Environment
- Operating System: mac os 10.14.6 (18G103)
- JDK version: jdk-11.0.4.jdk
- Visual Studio Code version: Version: 1.50.1
- Java extension version: v0.69.0

##### Steps To Reproduce
0.  clone my repo [https://github.com/clembou590/vscodeissue](https://github.com/clembou590/vscodeissue)
1. then in terminal go to folder named 'first' and run 'mvn clean install'
2. open a "vscode workspace" that contains ONLY folder named second. (THIS IS IMPORTANT OTHERWISE THERE IS NO BUG)
3. open file SecondApplication.java then on `MySharedEnum.VSCODE;` do F12 _(go to definition)

--> FIRST ISSUE:
When trying to search references for VSCODE(0) nothing is found (even though the enum VSCODE is used in SecondApplication.java ) 

--> SECOND ISSUE:
the values are incorrect 
on the left vscode should not show VSCODE(0) but VSCODE(2)
Screenshot 2020-10-21 at 17 29 58

### PR DESCRIPTION
**Pull request by snjeza**
Fixes #1665 
Requires https://github.com/eclipse/eclipse.jdt.ls/pull/1632

Signed-off-by: Snjezana Peco <snjezana.peco@redhat.com>

### CODE REVIEW

**Comment by snjeza** [comment]
cc @rgrunber

File: README.md [hunk 1]
```diff
@@ -184,6 +184,7 @@ The following settings are supported:
 
 New in 0.74.0:
```
Comment (rgrunber) [inline README.md:185]: I'd create a New in 0.77.0 and then we can just merge this in.

Comment (snjeza) [reply README.md]: I have updated the PR.

Comment (rgrunber) [reply README.md]: Thanks. Feel free to merge this.

**Pull request merged**